### ISSUE TITLE
Additional simulator controls

### ISSUE THREAD

**Issue by microbit-robert** [microsoft/pxt-microbit#6784]
**Is your feature request related to a problem? Please describe.**
The touch logo on the micro:bit simulator currently acts as both a button and a slider for mouse and keyboard users. However, an HTML role does not exist for screen reader users to encompasses both behaviours. Ideally, the touch logo should be role="button" and the compass should be a separate slider control with role="slider" and accompanying attributes.

It also not currently possible for keyboard users to tilt the micro:bit simulator. Mouse users can achieve this by hovering over the board. This limits keyboard users to simulating shake gestures only.

**Describe the solution you'd like**
A separate keyboard control / element for the compass that implements slider attributes and behaviours with the touch logo implementing only button roles and behaviours. There may be a way to implement this without significantly impacting the current visual design.

A control, that when focused (possibly the board itself?), allows the arrow keys to tilt the simulator. Releasing the arrow keys should cause the micro:bit to return to its default position. The control needs to have enough visual representation that keyboard users will understand what it does, and the simulator aria-live region and an aria-label can be used to provide feedback to screen reader users.

**Describe alternatives you've considered**
None. The touch logo could potentially work as two separate roles, but this breaks down when it needs to be both.

**Additional context**
Not expected for 2026 release.
